### PR TITLE
Add core files.

### PR DESCRIPTION
**Pull request by dlongley**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,10 @@
+*.py[co]
```
Comment (davidlehn) [inline .gitignore]: Not using python here.

File: WrmHintList.vue [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    };
+  },
+  methods: {
+    async select(hint) {
+      this.selectedHint = hint;
+      try {
+        // wait for selection to be handled
+        await this.onSelect(hint);
+      } catch(e) {
+        console.error(e);
```
Comment (davidlehn) [inline WrmHintList.vue]: Is catching errors intentional or debugging?

File: WrmHintChooser.vue [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+      const self = this;
+      await requestStorageAccess();
+      this.needsStorageAccess = false;
+      let promise = Promise.resolve();
+      this.$emit('load-hints', {
+        waitUntil: p => promise = p
+      });
+      try {
+        await promise;
+        console.log('promise waited');
```
Comment (davidlehn) [inline WrmHintChooser.vue]: Debugging?

File: WrmHintList.vue (hunk 2 above)
Comment (dlongley) [reply WrmHintList.vue]: Intentional in this case.

**Pull request merged**